Source/UnitTests/UnitTests.vcxproj

```diff
@@ -19,13 +19,14 @@
   <ItemDefinitionGroup>
     <!--This project also compiles gtest-->
     <ClCompile>
-      <AdditionalIncludeDirectories>$(ExternalsDir)gtest\googletest\include;$(ExternalsDir)gtest\googletest;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(ExternalsDir)gtest\googletest\include;$(ExternalsDir)gtest\googletest;$(ExternalsDir)\rcheevos;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
It would be better to use `<Import Project="$(ExternalsDir)rcheevos\exports.props" />`. (See PR #13534)